lampo-bdk-wallet/src/lib.rs

```diff
@@ -198,37 +220,17 @@ impl WalletManager for BDKWalletManager {
         };
         let wallet = self.wallet.borrow();
         let mut wallet = wallet.lock().unwrap();
-        let client = bdk_esplora::esplora_client::Builder::new(esplora_url).build_blocking()?;
-        let checkpoints = wallet.latest_checkpoint();
-        let spks = wallet
-            .spks_of_all_keychains()
-            .into_iter()
-            .map(|(k, spks)| {
-                let mut first = true;
-                (
-                    k,
-                    spks.inspect(move |(_spk_i, _)| {
-                        if first {
-                            first = false;
-                        }
-                    }),
-                )
-            })
-            .collect();
+        let sync_request = wallet.start_sync_with_revealed_spks(); // .inspect
         log::info!("bdk start to sync");
 
-        let (update_graph, last_active_indices) =
-            client.scan_txs_with_keychains(spks, None, None, 50, 2)?;
-        let missing_heights = wallet.tx_graph().missing_heights(wallet.local_chain());
-        let chain_update = client.update_local_chain(checkpoints, missing_heights)?;
-        let update = Update {
-            last_active_indices,
-            graph: update_graph,
-            chain: Some(chain_update),
-        };
+        let client = bdk_esplora::esplora_client::Builder::new(esplora_url).build_blocking();
```

Comment:
The point of using bdk alpha and beta was to use the bitcoin core backend

In the issue https://github.com/vincenzopalazzo/lampo.rs/issues/317 there is the following assertion 

>Keep in mind that Lampo currently uses Bitcoin Core (or similar tools) to fetch information from the network.